opl/investigator/elasticsearch_loader.py

```diff
@@ -1,10 +1,15 @@
 import json
 import logging
 import tempfile
+from requests.auth import HTTPBasicAuth
 
 import opl.http
 import opl.status_data
 
+# Environment variables for OpenSearch credentials
+open_search_username = "insights_perf"
+open_search_password = os.environ.get('OPEN_SEARCH_PASSWORD')
```

Comment:
I would add name of the variable to investigator config just to force user to understand it is needed.